services/storage/src/simcore_service_storage/core/settings.py

```diff
@@ -24,7 +18,7 @@
 
 class ApplicationSettings(BaseApplicationSettings, MixinLoggingSettings):
     STORAGE_HOST: str = "0.0.0.0"  # nosec
```

Comment:
earlier you have taken the hostname from the shell env-var `HOSTNAME`, here you use `0.0.0.0` always. In the rare case that there is a networking issue (inside docker or inside the cloud VPC, whatever), having slightly different things might lead to very weird issues. I suggest harmonizing this and settling on one single place the host is taken from.